packages/maskbook/src/components/shared/qrcode.tsx

```diff
@@ -74,10 +74,15 @@ export function QRCode({ text, options = {}, canvasProps }: QRProps) {
     )
 }
 
-export function WKWebkitQRScanner(props: { onScan?: (val: string) => void; onQuit?: () => void }) {
+export function NativeQRScanner(props: { onScan?: (val: string) => void; onQuit?: () => void }) {
     useAsync(async () => {
         try {
-            props.onScan?.(await iOSHost.scanQRCode())
+            if (nativeAPI?.type === 'iOS') {
+                props.onScan?.(await nativeAPI.api.scanQRCode())
+            } else {
+                // TODO:
```

Comment:
use native qr scanner for android platform? (future)

Reply:
Yes, I leave it here for that purpose.